packages/styles/src/components/list.scss

```diff
@@ -0,0 +1,54 @@
+@use './../themes/bootstrap/core' as *;
```

Comment:
We're going to remove bootstrap, this import can't stay here. Please replace every reference to bootstrap with internal functionality.

Reply:
It is removed.